windows/progress-view/ProgressViewViewManager.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "ProgressViewViewManager.h"
+#include "NativeModules.h"
+#include "ProgressViewView.h"
+
+namespace winrt {
+    using namespace Microsoft::ReactNative;
+    using namespace Windows::Foundation::Collections;
+    using namespace Windows::UI::Xaml;
+}
+
+namespace winrt::progress_view::implementation {
+
+    ProgressViewViewManager::ProgressViewViewManager() {}
+
+    // IViewManager
+    winrt::hstring ProgressViewViewManager::Name() noexcept {
+        return L"RNCProgressViewWindows";
+    }
+
+    winrt::FrameworkElement ProgressViewViewManager::CreateView() noexcept {
+        return winrt::progress_view::ProgressViewView(m_reactContext);
+    }
+
+    // IViewManagerWithReactContext
+    winrt::IReactContext ProgressViewViewManager::ReactContext() noexcept {
+        return m_reactContext;
+    }
+
+    void ProgressViewViewManager::ReactContext(IReactContext reactContext) noexcept {
+        m_reactContext = reactContext;
+    }
+
+    // IViewManagerWithNativeProperties
+    IMapView<hstring, ViewManagerPropertyType> ProgressViewViewManager::NativeProps() noexcept {
+        auto nativeProps = winrt::single_threaded_map<hstring, ViewManagerPropertyType>();
+        nativeProps.Insert(L"progress", ViewManagerPropertyType::Number);
+        nativeProps.Insert(L"progressTintColor", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"trackTintColor", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"progressViewStyle", ViewManagerPropertyType::String);
+        nativeProps.Insert(L"progressImage", ViewManagerPropertyType::String);
```

Comment:
Images will get passed using the 'require()' call from the JS side, which produces a JSON. Change ::String to ::Map.
(ImageViewManager.cpp for reference)